atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/OracleConnectionConfig.java

```diff
@@ -91,7 +91,7 @@ public String getDriverClass() {
     @Override
     @Value.Default
     public String getTestQuery() {
-        return "";
+        return "SELECT 1 FROM DUAL";
```

Comment:
It might be better to bring back https://github.com/palantir/atlasdb/pull/4602 and change the other DB connection configs to use an empty string by default.

It looks like the only reason we reverted this in https://github.com/palantir/atlasdb/pull/4617 was because of bugs in the Oracle driver.

But then we effectively brought this back in https://github.com/palantir/atlasdb/pull/6018, after which we use `isValid` to test connections.